internal/lib/sandbox/namespaces.go

```diff
@@ -303,9 +316,9 @@ func infraPid(infra *oci.Container) int {
 // calling a container.State() in batch operations
 func nsPathGivenInfraPid(ns NamespaceIface, nsType NSType, infraPid int) string {
 	// caller is responsible for checking if infraContainer
-	// is valid. If not, infraPid should be negative
+	// is valid. If not, infraPid should be less than or equal to 0
 	if ns == nil || ns.Get() == nil {
-		if infraPid >= 0 {
+		if infraPid > 0 {
```

Comment:
this doesn't match the new comment on 319.  Did you mean to remove the `=` here?

Reply:
the comment is meant to say "if it's not valid". is the wording unclear?

Reply:
I also had to read this part twice, and it is correct, including the comment, but maybe not super clear. Not sure how to improve it though :(